site/src/components/AppLink/AppLink.stories.tsx

```diff
@@ -41,14 +44,16 @@ export const HealthInitializing = Template.bind({})
 HealthInitializing.args = {
   username: "developer",
   workspaceName: MockWorkspace.name,
-  appName: "code-server",
+  appSlug: "code-server",
+  appName: "Code Server",
   health: "initializing",
 }
 
 export const HealthUnhealthy = Template.bind({})
 HealthUnhealthy.args = {
   username: "developer",
   workspaceName: MockWorkspace.name,
-  appName: "code-server",
+  appSlug: "code-server",
+  appName: "Code Server",
```

Comment:
I think it'd be better to call this `appDisplayName` instead. Including in the Terraform, it'd be best to deprecate `name` for `display_name`, and require `slug`.